proto/iamanager/v5/iamanager.proto

```diff
@@ -70,13 +71,13 @@ message NodeAttribute {
 }
 
 message NodeInfo {
-    string                 id         = 1;
-    string                 type       = 2;
+    string                 node_id    = 1;
+    string                 node_type  = 2;
     string                 name       = 3;
```

Comment:
should we rename name -> node_name add other filed as well
If not, what is the goal of this renaming?